gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -82,13 +97,16 @@ resources:
               export OS_PROJECT_ID="demo_project_id"
               export OS_PASSWORD="demo_password"
 
-              # the following lines don't need to be changed
+              # set other region in NAME and URL if required
+              # available regions: cbk, dbl
```

Comment:
Let's not mention DBL before it goes live.

Reply:
removed